soccer/planning/RRTPlanner.cpp

```diff
@@ -183,10 +186,30 @@ InterpolatedPath* RRTPlanner::optimize(
 
         if (!changed) span++;
     }
-    // Done with the path
-    return cubicBezier(path, obstacles, motionConstraints, vi, vf);
-}
 
+    // Generate Cubic Bezier with Velocity Profile
+    vector<Point> points;
+    for (InterpolatedPath::Entry& entry : pts) {
+        points.push_back(entry.pos());
+    }
+    return generateCubicBezier(points, *obstacles, motionConstraints, vi, vf,
+                               smoothingIterations);
+}
```

Comment:
I'll see if we can add it to our style config, but it'd be nice to have newlines between function definitions :)